aksparse/client/aksparse/sparse.py

```diff
@@ -1,18 +1,31 @@
 from argparse import ArgumentError, ArgumentTypeError
 from typing import cast
 import arkouda as ak
+import os
+
+def set_arkouda_init(flag_val):
+    """If set to 1, certain functions will use Arkouda objects/functions
+      instead of work being done entirely on the Chapel server."""
+    if flag_val == True:
+        flag_val = 1
+    elif flag_val == False:
+        flag_val = 0
+    if flag_val != 1 and flag_val != 0:
+        print("Please enter either a 1 or a 0")
+    else:
+        os.environ.setdefault('ARKOUDA_INIT', str(flag_val))
```

Comment:
I would be careful with setting and accessing a environment variable like this in Python. Any variables set in python will not stick around like they will if `export ARKOUDA_INIT=flag_val` is run in terminal. I would recommend some type of config or global scope python variable here instead. Also, I would recommend changing the name, `ARKOUDA_INIT` is not very descriptive of what this is actually doing. Maybe something like `AK_PROC_METHOD`